RetroBar/Languages/français.xaml

```diff
@@ -22,9 +22,11 @@
     <s:String x:Key="allow_font_smoothing">_Autoriser le lissage des polices</s:String>
     <s:String x:Key="collapse_tray_icons">_Réduire les icônes de la zone de notification</s:String>
     <s:String x:Key="show_clock">Afficher l'_horloge</s:String>
+    <s:String x:Key="show_multi_mon">Show on _multiple displays</s:String>
```

Comment:
@VoXaN24, if you don't mind, could you help with the translation of a couple of lines for the release?

Reply:
<s:String x:Key="show_multi_mon">Affichez sur plusieur écrans</s:String>

Ps: Did u mean screen or the Windows desktop when u say display ?

Reply:
It refers to the screens, I think this translation is correct, I will use it, and as it is an option it will have the access key and I will correct the verb to present infinitive.
Like this: **Afficher sur _plusieurs écrans**

Thanks, now we only need to translate the line 29 that says "_Use software rendering".

Reply:
> 

Ok so you can use : "Afficher sur _plusieurs écrans"

_Use software rendering -> Utiliser le rendu logiciel

Reply:
Thank you very much, we are done, I will use your translations.